pkg/client/client.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+import (
+	"fmt"
```

Comment:
well it could be dropped, but I prefer to leave it there, will have a use in a mid term future

Reply:
I think that carrying unused/dead code in the repository should be avoided. It might get broken without anybody noticing, and, for newcomers (like me) it makes reading through and understanding the code base harder than it should.

Thus, I'd suggest to drop this from here and add it when it gets used. I you don't want to store this in your personal fork only, I think we could carry this in a separate devel branch in the main repo. WDYT?